### PR TITLE
[SSL] Restructured total tls info + provide more actionable items for error messages

### PR DESCRIPTION
**Pull request by kodster28**
Partially addresses PCX-5607

### CODE REVIEW

File: content/dns/_partials/_onboard-warning.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+When you [add a domain](/fundamentals/get-started/setup/add-site/) to Cloudflare, Cloudflare protection will be in a [pending state](/dns/zone-setups/reference/domain-status/) until we can verify ownership. This could take up to 24 hours to complete.
```
Comment (kodster28) [inline content/dns/_partials/_onboard-warning.md:8]: Just moving an existing statement into a partial

File: content/ssl/_partials/_ussl-limitations-table.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+By default, Cloudflare [Universal SSL certificates](/ssl/edge-certificates/universal-ssl/) only cover your root domain and one level of subdomain.
```
Comment (kodster28) [inline content/ssl/_partials/_ussl-limitations-table.md:8]: Moving existing statement to a partial.

File: content/ssl/edge-certificates/additional-options/total-tls/_index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+pcx_content_type: concept
+title: Total TLS
+weight: 14
+---
+
+# Total TLS
```
Comment (kodster28) [inline content/ssl/edge-certificates/additional-options/total-tls/_index.md:7]: Moving existing content to a new page.

File: content/ssl/_partials/_ussl-limitations-solutions.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+To prevent insecure connections on a multi-level subdomain, do one of the following:
+
+- Enable [Total TLS](/ssl/edge-certificates/additional-options/total-tls/), which automatically issues new certificates to covered any proxied hostnames not covered by a Universal certificate.
```
Comment (ranbel) [inline content/ssl/_partials/_ussl-limitations-solutions.md]:
```suggestion
- Enable [Total TLS](/ssl/edge-certificates/additional-options/total-tls/), which automatically issues new certificates to any proxied hostnames not covered by a Universal certificate.
```

**Review by ranbel — APPROVED** [review]

**Pull request merged**